images/installer/Dockerfile.upi.ci

```diff
@@ -45,13 +45,12 @@ ENV CLOUDSDK_PYTHON=/usr/bin/python
 
 RUN python3 -m pip install yq
 
-ENV TERRAFORM_VERSION=0.12.24
+ENV TERRAFORM_VERSION=1.0.11
 RUN curl -O https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
     unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /bin/
-ENV MATCHBOX_PROVIDER_VERSION=v0.3.0
-RUN curl -L -O https://github.com/poseidon/terraform-provider-matchbox/releases/download/${MATCHBOX_PROVIDER_VERSION}/terraform-provider-matchbox-${MATCHBOX_PROVIDER_VERSION}-linux-amd64.tar.gz && \
-    tar xzf terraform-provider-matchbox-${MATCHBOX_PROVIDER_VERSION}-linux-amd64.tar.gz && \
-    mv terraform-provider-matchbox-${MATCHBOX_PROVIDER_VERSION}-linux-amd64/terraform-provider-matchbox /bin/terraform-provider-matchbox
+ENV MATCHBOX_PROVIDER_VERSION=v0.5.0
+RUN curl -L -O https://github.com/poseidon/terraform-provider-matchbox/releases/download/v${MATCHBOX_PROVIDER_VERSION}/terraform-provider-matchbox-${MATCHBOX_PROVIDER_VERSION}-linux_amd64.zip  && \
+    unzip terraform-provider-matchbox-${MATCHBOX_PROVIDER_VERSION}-linux-amd64.zip -d /bin/
```

Comment:
ENV MATCHBOX_PROVIDER_VERSION=0.5.0
RUN curl -L -O https://github.com/poseidon/terraform-provider-matchbox/releases/download/v${MATCHBOX_PROVIDER_VERSION}/terraform-provider-matchbox_${MATCHBOX_PROVIDER_VERSION}_linux_amd64.zip  && \
    unzip terraform-provider-matchbox_${MATCHBOX_PROVIDER_VERSION}_linux_amd64.zip -d /bin/

Reply:
Sorry, I'm not able to fix this with my IDE due to some odd spaces being added by it and have been eyeballing this with a text editor. I'll fix this properly

Reply:
No worries. Pay attention to the underscores and the ${MATCHBOX_PROVIDER_VERSION} use; one place uses the prefix 'v' and another doesn't. The lines I added above in the comments work correctly.

Reply:
We should skip most of these tests as well. Only the image builder and UPI metal tests need to pass, IMO.